utils/fuzzer-options/examples/qemu.rs

```diff
@@ -0,0 +1,31 @@
+use fuzzer_options::{parse_args, Commands};
+use std::env;
+use std::path::{Path, PathBuf};
+
+use libafl_qemu::Emulator;
+
+fn fuzz_with_qemu(_: &[PathBuf], qemu_args: &[String]) {
+    let env: Vec<(String, String)> = env::vars().collect();
```

Comment:
```suggestion
    env::remove_var("LD_LIBRARY_PATH");
    let env: Vec<(String, String)> = env::vars().collect();
```

cargo sets this variable when using `cargo run`, it could cause an issue when running the emulator

Reply:
fixed

Reply:
that was probably my bad lol